features/support/registration_helpers.rb

```diff
@@ -22,6 +22,7 @@ def add_submitted_registration(registration, load_root_page = true)
 end
 
 def add_unsubmitted_registration(registration, load_root_page = true)
+  # Possible issue: this takes user through front end flow even if called from back office flow.
```

Comment:
What's the issue @andrewhick ? If you don't pass in `false` for the `load_root_page` then yep, the first thing it will do is load the front office start page.

However as long as your step has taken you to the start page in the back office (for example by clicking the new reg link) and you pass in `false` it shouldn't switch you over.

Happy to be shown otherwise however, in which case let me know and I can fix it.

Reply:
Sorry, this was an old comment before I fully understood `load_root_page`. It now seems to be taking a user through the correct flow as intended. I'll remove the comment.